docs/Makefile

```diff
@@ -19,6 +19,10 @@ build: build-examples build-helm
 	mkdir -p site/downloads/helm
```

Comment:
This `build` target needs to depend on a `build-proto` target.

Lol, while you are at it, please fix the `.PHONE: build-helm` to be `.PHONY: build-helm`.  Just saw that.